tests/functional/test_create_sync_folder.py

```diff
@@ -10,12 +10,14 @@ def test_create_folder(manager_factory, tmp_path):
     test_processor = Processor(engine, "")
     dp = DocPair
     dp.local_path = tmp_path
+    dp.remote_ref = "defaultSyncRootFolderItemFactory"
+    dp.folderish = True
     create_rem = test_processor._create_remotely
+
     full_path = create_rem(dp, dp, "default - domain - Workspaces - Administrator")
     assert str(path.join(tmp_path, "Administrator")) == str(full_path)
-    full_path_duplicated = create_rem(
-        dp, dp, "default - domain1 - Workspaces - Administrator"
-    )
+
+    full_path = create_rem(dp, dp, "default - domain1 - Workspaces - Administrator")
```

Comment:
What is the change here?

Reply:
we are using same variable to check multiple cases. because after first assert we do not need the variable again. it will take less memory.